### ISSUE TITLE
Not saving data frames

### ISSUE THREAD

**Issue by vartika271987** [s7u512/NSSO-77-SAS#2]
Hello! I tried to extract the data files using your code. However, I get stuck in the 2nd R Script that throws no error per se, but also does not save the data frames as .R or .csv files.
Here's a snapshot of the R environment 
<img width="298" alt="image" src="https://github.com/s7u512/NSSO-77-SAS/assets/51452385/f909a8fe-39e4-406c-a092-31e21005024c">

I get these warnings when I am multiprocessing the data frames: 
<img width="386" alt="image" src="https://github.com/s7u512/NSSO-77-SAS/assets/51452385/4b5d6a5e-1cd9-4155-bc19-c99bb2177db5">
 but I do see the data frames that have these values: 
<img width="336" alt="image" src="https://github.com/s7u512/NSSO-77-SAS/assets/51452385/d8678155-e55a-4e8c-832c-1b920c0d5bfd">

Any help is appreciated!

**Comment by s7u512** [comment]
I think you are in the wrong repo :) (It was my mistake, I had a wrong link)

From  your screenshots, I am guessing it is about [this](https://github.com/s7u512/NSSO_HCES_2022-23/) repo.

Anyways, I have made some updates there. Please try again. 

The screenshot of the environment looks as it should at line 90.

The 3 warnings as a result of the multiprocessing can be safely ignored.

Your third screenshot shows that the data has been extracted into a single dataframe. Line 90 through 93 should seperate them out. It might take a while depending on your system's capacity. 

Alternatively, I have now added some commented out code you can use in lieu of the parallel processing. 

Either way I suggest you run the updated script again, and let me know if it was solved.

For consistency, I request you click [here](https://github.com/s7u512/NSSO_HCES_2022-23/issues/new) if the issue persists, so that I can track it properly.

Once again, it was my bad for providing the wrong link in the `Readme` earlier.

**Comment by vartika271987** [comment]
Thank you for your prompt response and for making the changes. I realized that I had not run the lines 90-93, as they were in the `alternative` block of code and without looking into it much I thought since I am doing the multi-processing, I did not need to run the code in the alternative block. I have now taken the liberty to update the code a bit to ensure other readers clearly know which sections to use or skip. Hope that is okay!

It all works very well now and I can see the data - thanks a lot for this public good!!
